dataflow/build.xml

```diff
@@ -162,8 +161,7 @@
                  additionalParam="-Xdoclint:all,-missing"
                  failonerror="true"
                  executable="${javadoc.bin}"
-                 classpath="${build}:${javacutil.lib}:${javac.lib}:${javadoc.lib}:${java.home}/../lib/tools.jar"
-                 bootclasspath="${javac.lib}:${javadoc.lib}:${java.home}/lib/rt.jar">
+                 classpath="${build}:${javacutil.lib}:${javac.lib}:${javadoc.lib}:${java.home}/../lib/tools.jar">
```

Comment:
Can you see whether the `tools.jar` can be removed, here and in `javacutil/build.xml`? javac and javadoc should be the only required dependencies.
Also, `checker/build.xml` mentions `tools.jar` in a comment, but then is not used in the javadoc target.
Hopefully this can be simplified a bit.